packages/contracts/contracts/L2/predeploys/OVM_GasPriceOracle.sol

```diff
@@ -13,14 +17,20 @@ import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
  *
  * Runtime target: OVM
  */
-contract OVM_GasPriceOracle is Ownable {
+contract OVM_GasPriceOracle is Ownable, iOVM_GasPriceOracle {
 
     /*************
      * Variables *
      *************/
 
-    // Current l2 gas price
+    // Current L2 gas price
     uint256 public gasPrice;
+    // Current L1 base fee
+    uint256 public l1BaseFee;
+    // Amortized cost of batch submission per transaction
+    uint256 public overhead = 2750;
+    // Value to scale the fee up by
+    uint256 public scalar = 1;
 
```

Comment:
TODO: add scalar decimals

Reply:
For some reason this isn't set to 1 at startup. Each time that I start the `docker-compose` setup and `eth_call` this, it first returns `0` until I manually set it in the tests